apps/service-portal/src/index.html

```diff
@@ -37,6 +37,12 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script
+      async
+      defer
+      data-domain="beta.minarsidur.island.is"
```

Comment:
Should we not be using the new domain here?

Reply:
Yep